go/libraries/doltcore/mvdata/data_mover.go

```diff
@@ -169,17 +170,19 @@ func NewDataMover(ctx context.Context, root *doltdb.RootValue, fs filesys.Filesy
 	return imp, nil
 }
 
-func (imp *DataMover) Move(ctx context.Context) error {
+func (imp *DataMover) Move(ctx context.Context) (int64, error) {
```

Comment:
You should document the new return param

Reply:
done